### PR TITLE
Heading style

### PR DESCRIPTION
**Pull request by ldez**
The recent changes breaks the heading style with some theme.

Currently (dev mode):
capture du 2016-05-07 16-45-56

With my changes:
capture du 2016-05-07 16-46-26

I have try with several themes.

### CODE REVIEW

**Comment by nicorikken** [comment]
I just noticed the same thing when switching themes. Good catch, good fix

**Comment by nicorikken** [comment]
You didn't refresh or commit your generate language, that's why your tests succeed 😉

**Comment by ldez** [comment]
mistake !!!!

**Comment by ldez** [comment]
Fixed  :disappointed_relieved:

**Comment by nicorikken** [comment]
Almost... 😄

**Comment by ldez** [comment]
really fixed  :cold_sweat:

File: spec/partials/titles-grammar-spec.coffee [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe 'Titles', ->
+fdescribe 'Titles', ->
```
Comment (nicorikken) [inline spec/partials/titles-grammar-spec.coffee]: Typo?

**Comment by ldez** [comment]
Note I use `.toEqualJson` insteaf of `.toEqual` for a better tests failure report

File: spec/partials/titles-grammar-spec.coffee [hunk 2]
```diff
@@ -23,9 +23,9 @@ describe 'Titles', ->
       marker = Array(equalsSigns + 1).join('=')
       {tokens} = grammar.tokenizeLine "#{marker} Heading #{level}"
       expect(tokens).toHaveLength 3
-      expect(tokens[0]).toEqual value: "#{marker}", scopes: ['source.asciidoc', "markup.heading-#{level}.asciidoc", 'markup.heading.marker.asciidoc']
-      expect(tokens[1]).toEqual value: " ", scopes: ['source.asciidoc', "markup.heading-#{level}.asciidoc", 'markup.heading.space.asciidoc']
-      expect(tokens[2]).toEqual value: "Heading #{level}", scopes: ['source.asciidoc', "markup.heading-#{level}.asciidoc"]
+      expect(tokens[0]).toEqualJson value: "#{marker}", scopes: ['source.asciidoc', "markup.heading.heading-#{level}.asciidoc", 'markup.heading.marker.asciidoc']
```
Comment (nicorikken) [inline spec/partials/titles-grammar-spec.coffee]: is `.toEqualJson` safer than `.toEqual` ?

Comment (ldez) [inline spec/partials/titles-grammar-spec.coffee]: :smile:

**Comment by ldez** [comment]
I have added a small comment documentation about the use of `f` with `fdescribe` (focused `describe`) or `fit` (focused `it`)

**Comment by nicorikken** [comment]
Nice, so I learned something new today. Good PR, merge!

**Pull request merged**